sdk/formrecognizer/Azure.AI.FormRecognizer/src/DataTableCell_internal.cs

```diff
@@ -8,5 +8,7 @@ namespace Azure.AI.FormRecognizer.Models
     [CodeGenModel("DataTableCell")]
     internal partial class DataTableCell_internal
     {
+        /// <inheritdoc />
+        public float? Confidence { get; }
```

Comment:
Per discussion on our internal code gen thread, let's override the Deserialize method instead of marking as a `float?`.

Reply:
thanks for the guidance here!
Code updated